### PR TITLE
Fgts 84 create health bug

### PR DESCRIPTION
**Pull request by jDenyes**

### CODE REVIEW

File: godotproject/Player.gd [hunk 1]
```diff
@@ -503,7 +503,7 @@ func apply_knockback(knockback_up_only := false):
 	
 	# Play Ouch sound effect
 	$OuchSoundPlayer.play()
-
+
```
Comment (charasyn) [inline godotproject/Player.gd]:
```suggestion

```

Comment (charasyn) [reply godotproject/Player.gd]: I think this was just accidentally added

**Review by charasyn — COMMENTED** [review]
Can you do a bit of cleanup? Move all the health bug related files into Objects/HealthBug and also it looks like there are two PNG files but only one is used? Other than that looks good.

File: godotproject/Player.gd [hunk 2]
```diff
@@ -526,14 +526,25 @@ func death_animation_over():
...
 	# Increment star piece counter
 	star_piece_count += 1
-	print("Star piece count: ", star_piece_count)
 	# Play sound
 	$CollectStarPieceSoundPlayer.play(0)
 	# Delete star piece
 	star_piece.queue_free()
+	
+func collect_health_bug(health_bug):
+	if (health < 5):
```
Comment (fheysel) [inline godotproject/Player.gd]: I would change this to a player const HEALTH_MAX, easier for hard/easy mode implementation

File: godotproject/Player.gd [hunk 3]
```diff
@@ -526,14 +526,25 @@ func death_animation_over():
...
+		health += 1
+	# Play Sound : Temporary sound waiting for Health Bug Sound
+	$CollectStarPieceSoundPlayer.play(0)
```
Comment (fheysel) [inline godotproject/Player.gd]: can we get a different sound effect for the health bug? Maybe like a big gulp or something?

**Comment by fheysel** [comment]
A few changes. It might've been easier to make the bug to negative damage instead of making a whole separate collectables section. But if it works Im fine with it.

**Review by charasyn — APPROVED** [review]
Looks good to me

**Pull request merged**